src/index.html

```diff
@@ -24,26 +24,56 @@ <h1>What's Cookin'</h1>
     </header>
     <main>
       <section id="filter" class="filter hidden">
+        <!-- This is the section for the checkboxes -->
       </section>
-      <section id="recipeDisplay" class="recipe-display">
+      <section id="recipeDisplay" class="recipe-display hidden">
         <div class="recipe-card">
           <div>
           <img class="recipe-image" src="https://www.pumpkinnspice.com/wp-content/uploads/2020/08/creamy-macaroni-cheese-4.jpg"/>
         </div>
           <article>Delicious Mac n Cheese</article>
         </div>
       </section>
-      <section id="currentRecipeView" class="current-recipe-view hidden">
-        <h3>Wow a Delicious Recipe</h3>
-        <img src="https://www.pumpkinnspice.com/wp-content/uploads/2020/08/creamy-macaroni-cheese-4.jpg"/>
-        <section class="ingredients-list">
+      <section id="currentRecipeView" class="current-recipe-view">
+        <section class="current-recipe-name">
+          <h3>Loaded Chocolate Chip Pudding Cookie Cups</h3>
         </section>
-        <section class="instructions-list">
+        <section class="current-recipe-add-to-fav">
+          <button class="favorite-button">Favorite</button>
         </section>
-        <section class="tags-display">
+        <section class="current-recipe-img">
+          <img src="https://spoonacular.com/recipeImages/595736-556x370.jpg"/>
         </section>
-        <div class="save-buttons">
-          <button class="favorite-button">Favorite</button>
+        <section class="current-recipe-ing">
+          <ul>
```

Comment:
Great idea to build this out as a place holder! This will really help our formatting.